packages/notification-center/package.json

```diff
@@ -34,6 +34,7 @@
     "acorn": "^8.7.1",
     "babel-loader": "^8.2.4",
     "jest-transform-stub": "^2.0.0",
+    "lodash.clonedeep": "^4.5.0",
```

Comment:
Is it a devDependecy? Or a regular dep?

Reply:
it's a dev dep :) I use it to solve the issue with the theme prop I talked about here: https://github.com/novuhq/novu/pull/2156/files#r1038131817